### ISSUE TITLE
Wrong filament after MMU power fault (3.8.0-RC2, 1.0.6)

### ISSUE THREAD

**Issue by vintagepc** [prusa3d/Prusa-Firmware#2159]
Started getting MMU power faults (all 5 MMU LEDs blinking red and green). Instructions say to reset the MMU to correct. However, the MMU will re-home and park on filament 1, not the correct filament the T command had requested. It can be worked around if you're quick and can hit "change filament", but not ideal. 

(Note this happens only if the selector/FINDA is clear, of course if filament is sensed, it will not move the selector)

**Comment by MB3Y** [comment]
This problem #2159 seems related to my finding documented in #2476 .
May I ask if this problem is considered to be taken up any time soon? Original submission dates 6months back meanwhile and before I submitted I was checking for SW updates... bad luck however

**Comment by MB3Y** [comment]
Resubmitted in the correct GitHub repository under #2476

**Comment by JCaba** [comment]
I just had this same problem today:  I resolved as follows (maybe ulimately related to "spooljoin" during reset?

March 12 2020:
Was using MMU positions/filaments stations 1 and 3:
MMU indicated line 3 (soluble) was jammed, when in fact line 1 (PLA) was jammed in the print-head.  After extracting line 1 (by faking the machine into thinking line 1 was clear (by removing front PTFE tube and clearing the MMU, then pause MK3, remove side fan, increase head temp to 250, then extracting filament from print-head).
Then after resume, the MMU wanted to load line 5 – which wasn’t being used and was actually empty. (I faked it – by temporarily inserting filament so it was ‘loaded’) , then it wanted to load line 2 (faked it again).  Printer started to use line 2.
Paused, went to settings, turned off the ‘spooljoin’ function, reset MMU, Then the MMU/Mk3 resumed using the normal/selected filaments (line 1 and line 3).
